modules/prebidServerBidAdapter.js

```diff
@@ -0,0 +1,342 @@
+import Adapter from 'src/adapter';
+import bidfactory from 'src/bidfactory';
+import * as utils from 'src/utils';
+import { ajax } from 'src/ajax';
+import { STATUS, S2S } from 'src/constants';
+import { cookieSet } from 'src/cookie.js';
+import adaptermanager from 'src/adaptermanager';
+import { config } from 'src/config';
+import { VIDEO } from 'src/mediaTypes';
+import CONSTANTS from 'src/constants.json';
+
+const getConfig = config.getConfig;
+
+const TYPE = S2S.SRC;
+const cookieSetUrl = 'https://acdn.adnxs.com/cookieset/cs.js';
+let _synced = false;
+
+let _s2sConfigDefaults = {
+  enabled: false,
+  endpoint: CONSTANTS.S2S.DEFAULT_ENDPOINT,
```

Comment:
These constants are not defined in `constants.json`

Reply:
As mentioned elsewhere we should leave the endpoint defaults blank, and probably just hard code the adapter, either here or in `config.js` like was mentioned in another comment.